Example/ios/Podfile.lock

```diff
@@ -807,4 +807,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f37cea0ea1b6dc523fb88a9c2bdb9993f4b567ce
 
-COCOAPODS: 1.14.0
+COCOAPODS: 1.13.0
```

Comment:
Please bump CocoaPods to 1.14.0 or even 1.14.2. Here's the command: `brew upgrade cocoapods`.